client/src/pages/dashboard/Dashboard.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import {useEffect, useState} from 'react';
+import Row from '../../components/row/Row';
+import './Dashboard.css';
+import noIcon from '../../assets/Delete.svg';
+import yesIcon from '../../assets/Checked.svg';
+import SearchBar from '../../components/searchbar/SearchBar';
+
+const proposals = [
+  {
+      title: "some title",
+      voted: yesIcon
+  },
+
+  {
+      title: "Please give us money. Thanks",
+      voted: noIcon
+  },
+
+  {
+    title: "aljewnoinxansxlwnlndwondwdn"
+  },
+  
+  {
+    title: "We need help building XYZ",
+    voted: yesIcon
+  },
+
+  {
+    title: "some title",
+    voted: noIcon
+  },
+
+  {
+  title: "some title"
+  },
+  {
+    title: "We need help building XYZ",
+    voted: yesIcon
+  },
+
+  {
+    title: "some title",
+    voted: noIcon
+  },
+
+  {
+  title: "some title"
+  }
+]
+
+
+
+function Dashboard() {
+  const proposalHTML = []
+  for (let i = 0; i < proposals.length; i++) {
+    proposalHTML.push(<Row title={proposals[i].title} vote={proposals[i].voted ? proposals[i].voted : ""} />);
+  }
+
+  const [input, setInput] = React.useState("");
+  const [proposalListDefault, setProposalListDefault] = React.useState(proposals);
```

Comment:
is proposalListDefault meant to simply store all the proposals before filtering? If so, you can just use a normal variable for this (it doesn't have to be a state variable)